Podcast/FetchByIDEndpointRequests.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  FetchByIDEndpointRequests.swift
+//  Podcast
+//
+//  Created by Jack Thompson on 8/27/18.
+//  Copyright © 2018 Cornell App Development. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+class FetchSeriesForSeriesIDEndpointRequest: EndpointRequest {
+    
+    var seriesID: String
+    
+    init(seriesID: String) {
+        self.seriesID = seriesID
+        super.init()
+        
+        path = "/series/\(seriesID)/"
+        httpMethod = .get
+    }
+    
+    override func processResponseJSON(_ json: JSON) {
+        processedResponseValue = Cache.sharedInstance.update(seriesJson: json["data"]["series"])
+    }
+}
+
+class FetchEpisodesForSeriesIDEndpointRequest: EndpointRequest {
+    
+    var seriesID: String
+    var offset: Int
+    var max: Int
+    
+    init(seriesID: String, offset: Int, max: Int) {
+        
+        self.seriesID = seriesID
+        self.offset = offset
+        self.max = max
+        
+        super.init()
+        
+        path = "/podcasts/episodes/by_series/\(seriesID)/"
+        
```

Comment:
could group these 3 together without line breaks in between. Just a suggestion

Reply:
plus one this suggestion